libpal/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+enable_language(ASM_MASM)
```

Comment:
This file breaks if building on anything besides Windows + Visual Studio. As a temporary work around, make this small change:

```
if(${CMAKE_GENERATOR} MATCHES "Visual Studio")
    enable_language(ASM_MASM)
    
    add_library(libpal cpuid_x64.asm)
    set_target_properties(libpal PROPERTIES OUTPUT_NAME pal)
endif()

```